react/features/large-video/components/LargeVideo.web.js

```diff
@@ -85,8 +90,11 @@ class LargeVideo extends Component<Props> {
  */
 function _mapStateToProps(state) {
     const testingConfig = state['features/base/config'].testing;
+    const { isOpen: shiftRight } = state['features/chat'];
```

Comment:
Can you build the class names in the render method instead? Pass some chatOpen flag on mapStateToProps

Reply:
Done